pkg/auth/none_authz_handler.go

```diff
@@ -0,0 +1,35 @@
+package auth
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/openshift/assisted-service/internal/identity"
+	"github.com/openshift/assisted-service/pkg/ocm"
+	"gorm.io/gorm"
+)
+
+/* NoneHandler is the authorizer middleware that is being used for
+   non-RHSSO authentication cases. It will basically authorize any
+   request, as there is no user and tenancy based concepts in these
+   cases
+*/
+type NoneHandler struct {
+}
+
+func (*NoneHandler) CreateAuthorizer() func(*http.Request) error {
+	return func(*http.Request) error {
+		return nil
+	}
+}
+
+func (*NoneHandler) OwnedBy(ctx context.Context, db *gorm.DB) *gorm.DB {
+	if identity.IsAdmin(ctx) {
```

Comment:
the tests for this logic is in authz_handler_test, right?

Reply:
Yes